run/README.md

```diff
@@ -0,0 +1,86 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Run Ruby Samples
+
+[Cloud Run][run_docs] runs stateless [containers](https://cloud.google.com/containers/) on a fully managed environment or in your own GKE cluster.
+
+## Samples
+
+|                 Sample                  |        Description       |     Deploy    |
+| --------------------------------------- | ------------------------ | ------------- |
+|[Hello World][helloworld]  | Quickstart | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30"/>][run_button_helloworld] |
+
+For more Cloud Run samples beyond Ruby, see the main list in the [Cloud Run Samples repository](https://github.com/GoogleCloudPlatform/cloud-run-samples).
+
+## Setup
+
+1. [Set up for Cloud Run development](https://cloud.google.com/run/docs/setup)
+
+2. Clone this repository:
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/ruby-docs-samples.git
+    ```
+
+## How to run a sample locally
+
+1. [Install docker locally](https://docs.docker.com/install/)
+
+2. [Build the sample container](https://cloud.google.com/run/docs/building/containers#building_locally_and_pushing_using_docker):
+
+    ```sh
+    export SAMPLE=$sample
+    cd $SAMPLE
+    docker build --tag $sample .
+    ```
```

Comment:
Are these instructions correct?

```sh
export SAMPLE=$sample
cd $SAMPLE
docker build --tag $sample .
invalid argument "." for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
```

Reply:
Probably needs to be `helloworld`.

Reply:
I guess I need to replace the "$sample" with my custom sample name?

Reply:
This is consistent with our other parent READMEs, but will add more comments.